doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2385,4 +2385,17 @@ Scripts that implement this association for specific service protocols are maint
 
 ``curl -X PUT --upload-file cvoc-conf.json http://localhost:8080/api/admin/settings/:CVocConf``
 
+.. _:AllowedCurationLabels:
+
+:AllowedCurationLabels
+++++++++++++++++++++++
+ 
+A JSON Object containing lists of allowed labels (up to 32 characters, spaces allowed) that can be set, via API or UI by users with the permission to publish a dataset. The set of labels allowed 
+for datasets can be selected by a superuser - via the Dataverse collection page (Edit/General Info) or set via API call. 
+The labels in a set should correspond to the states in an organization's curation process and are intended to help users/curators track the progress of a dataset through an externally defined curation process. 
```

Comment:
We probably don't need the word "externally" here. I get that integration is possible, but it's not required.

Reply:
I removed it. FWIW: The word is supposed to indicate that there really is no curation process going on in Dataverse - this is only a label and if you have to define it elsewhere, and enforce it's restrictions (such as what states can transfer to what state) elsewhere (manual or other app).